src/asm_ostream.cpp

```diff
@@ -551,6 +553,9 @@ int size(const Instruction& inst) {
     if (std::holds_alternative<LoadMapFd>(inst)) {
         return 2;
     }
+    if (std::holds_alternative<LoadMapAddress>(inst)) {
+        return 2;
+    }
     return 1;
 }
 
```

Comment:
This function is implemented twice. I think asm_ostream should use asm_marshal's version.